charts/gitops-server/README.md

```diff
@@ -0,0 +1,52 @@
+# Weave Gitops Helm Chart
+
+This is the [Weave Gitops](https://github.com/weaveworks/weave-gitops) [Helm](https://helm.sh) chart.
+
+It installs the weave-gitops-server component as a 1-replica deployment.
+
+Optionally it will also install:
+* Service Account
+* ClusterRoleBinding (to the service account) and ClusterRole with the
+  permissions required to run Gitops.
+* Service, this is optional as you may want to limit access to the UI to via
+  port-forwarding
+* Ingress
+* Test User -- A test user with hard-coded username & password with minimal
```

Comment:
The test user is actually a backup admin user in the case that OIDC is failing and cluster ops need to do things.
It is currently undergoing some feature realignment, see the thread here: https://weaveworks.slack.com/archives/C03244W0C8H/p1647967858091769

Reply:
Why would you have a backup admin with only read permissions?

Reply:
because those are all i added so far 🙃

Reply:
from a point of least privilige I'd like to only make it a requirement/recommendation when it's actually in a position to fulfill those extended duties ;)